can-detector/can-detector.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python3
```

Comment:
ROS will refuse to use this shebang line. If you convert this to a ROS node, use this shebang: `#!/usr/bin/env python`